lldb/source/Plugins/Process/scripted/ScriptedProcess.cpp

```diff
@@ -182,10 +182,16 @@ void ScriptedProcess::DidResume() {
   m_pid = GetInterface().GetProcessID();
 }
 
-Status ScriptedProcess::DoResume() {
+Status ScriptedProcess::DoResume(RunDirection direction) {
   LLDB_LOGF(GetLog(LLDBLog::Process), "ScriptedProcess::%s resuming process", __FUNCTION__);
 
-  return GetInterface().Resume();
+  if (direction == RunDirection::eRunForward) {
+    return GetInterface().Resume();
+  } else {
+    return Status::FromErrorStringWithFormatv(
```

Comment:
I don't expect you to support this, but can you put a 

// FIXME: Pipe reverse continue through Scripted Processes

here.  There's no reason why scripted processes couldn't support reverse continue.

Reply:
Done.